src/client/scripts/cardModal.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
+/* global Cards:false SnackbarMethods:false */
+
+Template.cardModal.helpers({
+    cardModalInfo: function(_id) {
+        return Cards.findOne({"_id": _id});
+    }
+});
+
+Template.cardModal.events({
+    "click #submitCommentButton": function(event){//eslint-disable-line
+        event.preventDefault();
+        var comment = event.target.form[0].value;
+        var image = null;
+
+        if(!comment || comment.length <= 4)
+            SnackbarMethods.DisplayMessage("Enter a more valuable comment",500);
+        else{
+            var author;
+
+            if(!Meteor.user())
+                author = Session.get("author");
+            else{
+                author = Meteor.user().profile.name;
+                image = UserMethods.getUserImage(Meteor.user()._id);//eslint-disable-line
+            }
+            var commentToAdd = new Comment().createdBy(author)
+              .withText(comment).createdAtTime(new Date()).withProfile(image);
+
+            Meteor.call("submitComment",this._id,commentToAdd);
+            event.target.form[0].value = "";
```

Comment:
0th item on form? Why not just 'event.target.comment.value = "";'
This is also used above, but I didn't notice it intially.
If another field were to be added to the form, this would confuse the hell outta of the person working with it.

Look in cardSubmitArea js and html on how to handle this